### ISSUE TITLE
SFRestRequest & /

### ISSUE THREAD

**Issue by jeffreyjackson** [forcedotcom/SalesforceMobileSDK-iOS#1016]
I am working on upgrading a client's application to use the newest mobile SDK.  The migration is going fine with the exception of 1 SFRestRequest I'm having issues with.  

A following instantiation no longer works:
```
SFRestRequest *request = [[SFRestRequest alloc] init];
```
Instead I have to do this:
```
 SFRestRequest *request = [SFRestRequest requestWithMethod:method path:path queryParams:@{}];
```
The above instantiation works fine.  However...

I have one bit of code that needs to pull down the user's profile picture.  Previously, this worked fine by doing something like this:

```
SFIdentityData *idData = [SFAuthenticationManager sharedManager].idCoordinator.idData;
SFRestRequest *request = [[SFRestRequest alloc] init];
request.endpoint = [idData.pictureUrl absoluteString];
request.method = SFRestMethodGET;
request.path = @"";
[[SFRestAPI sharedInstance] send:request delegate:_delegate];
```

Unfortunately, the ```[[SFRestRequest alloc] init]``` doesn't instantiate correctly so I'm resorting to the method ```requestWithMethod:path:queryParams:]```. Now I have a new issue:

With the original request (using 3.0 SDK) I would have something like this:
```
<SFRestRequest 0x7e340230 
endpoint: https://c.na19.content.force.com/profilephoto/005/F 
method: GET 
path:  
queryParams: [] 
>
```

Now, no matter what I do ```path``` is always prepending with a ```/``` in the front.  As a result the URL is always bad (using SDK 3.2 and original SFRestRequest statement).

```
<SFRestRequest 0x7cc8f090 
endpoint: (null) 
method: GET 
path: / 
queryParams: [] 
>
```

## tl;dr

I'm on API 33.0, with SDK 3.2.1.  I need an SFRestRequest to pull down the user's profile photo.  Whats the best method for this?

**Comment by jeffreyjackson** [comment]
cc @brad36

**Comment by qianwen** [comment]
@jeffreyjackson Hi I had the same issue as you. Did you solve the issue?

**Comment by jeffreyjackson** [comment]
i used SFRestRequests to get all the way to finding the URL, then I used AFNetworking from there.
